lib/utils/date.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:io' show Platform;
+import "package:intl/intl.dart";
+import 'package:fixnum/fixnum.dart';
+
+class DateUtils {
+  static final DateFormat _monthDateFormat = new DateFormat.Md(Platform.localeName);
+  static final DateFormat _yearMonthDayFormat = new DateFormat.yMd(Platform.localeName);
+  static final DateFormat _transactionDateFormat = new DateFormat.yMd(Platform.localeName).add_jm();
```

Comment:
In DateUtils let's not use "transaction..." it should be general (like formatMonthDate, formatYearMonthDay, etc...)